modules/network-observability-rate-limit-alert.adoc

```diff
@@ -0,0 +1,30 @@
+//
+// network_observability/configuring-operator.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-netobserv-dashboard-rate-limit-alerts_{context}"]
+= Creating Loki rate limit alerts for the NetObserv dashboard
+You can create custom alerts for the *Netobserv* dashboard to trigger alerts when Loki rate limits have been reached. 
```

Comment:
```suggestion
You can create custom rules for the *Netobserv* metrics to trigger alerts when Loki rate limits have been reached. 
```

@OlivierCazade what do you think ?

Reply:
Looks better. Thanks!

Reply:
You can create custom rules for the *Netobserv* dashboard metrics to trigger alerts when Loki rate limits have been reached.

Reply:
If you feel it makes more sense on the user perspective, why not